daemon/api_test.go

```diff
@@ -2335,10 +2335,14 @@ func (s *apiSuite) sideloadCheck(c *check.C, content string, head map[string]str
 		ensureStateSoonImpl(st)
 	}
 
+	mockedName := "local"
+	if expectedInstanceName == "" {
+		expectedInstanceName = mockedName
```

Comment:
IMHO it's better to explicitely pass "local" to all invocations of `sideloadCheck` (there are just a few of them) rather than this special-casing. But maybe I missed something, is there a specific reason to do it this way?